asapdiscovery-alchemy/asapdiscovery/alchemy/cli/alchemy.py

```diff
@@ -615,8 +648,32 @@ def predict(
     )
     console.print(message)
 
-    # use short switch to find if either value is not None
-    has_ref_data = reference_dataset or experimental_protocol
+    # check if we have a biological target
+    bio_target = target or result_network.target
+
+    # workout if we should upload to postera
+    if bio_target is not None and postera_molset_name is not None:
```

Comment:
Warn or log if one if `postera_molset_name` is present without `target`, ie someone probably wanted to upload but forgot to provide target. Currently doesn't give enough info about why no upload if skipped